### DISCUSSION TITLE
In-app explaination of habit strength

### DISCUSSION THREAD

**Discussion by adamvoss** [iSoron/uhabits#185] [Feature Requests]
Including information within the application regarding what the habit strength percentage means would make the information more accessible to new users and add polish to the app.

The percentages given on the charts screen have no intuitive meaning to a new user.  I could only really understand that higher numbers are better and I did not know what 100% was supposed to mean (I never set a goal after all, nor could I configure a goal).

In a roundabout way I found [the FAQ, which has an explanation](http://loophabits.org/faq.html).  I then noticed there is a "Help & FAQ" menu entry on the home screen to launch the FAQ webpage, though I suspect many users do not access it (especially users just previewing the app).

I don't see this as high priority, but it was a thought that occurred to me after I found the FAQ and then [this comment showing another user not understanding the percentage](https://github.com/iSoron/uhabits/issues/88#issuecomment-214309035) made me decide it worth creating the issue to be tracked.

**Comment by deutrino** [comment]
Or maybe don't call it a percentage at all? I had never read the faq until seeing this issue, but intuitively understood what was going on after using the app for a couple weeks; however, I think that many people would naturally think "percentage of what?" and maybe get caught up in wondering about it.
